scripts/dist.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
+
+import os
+from pathlib import Path
+import shutil
+import subprocess
+
+base = Path(os.getenv('MESON_DIST_ROOT'))
+
+os.unlink(base / '.gitattributes')
```

Comment:
You already have a Path object, skip importing os and just use `base.joinpath(".gitattributes").unlink()` or `(base / ".gitattributes").unlink()` but visually I don't like the second form as much, personal preference, I guess.
joinpath accepts multiple arguments, so `base.joinpath("m4", ".gitignore").unlink()` works too.
For recursively removing a subtree, I think shutil.rmtree is still the only stdlib way.